packages/calcite-components/src/components/shell/shell.tsx

```diff
@@ -38,6 +38,18 @@ export class Shell implements ConditionalSlotComponent {
    */
   @Prop({ reflect: true }) contentBehind = false;
 
+  //--------------------------------------------------------------------------
+  //
+  //  Event Listeners
+  //
+  //--------------------------------------------------------------------------
+
+  @Listen("calciteInternalShellPanelResize")
+  handleCalciteInternalShellPanelResize(event: CustomEvent<{ resizing: boolean }>): void {
+    this.panelIsResizing = !!event.detail?.resizing;
```

Comment:
I think you can drop the optional chaining operator since `detail` is guaranteed for this custom event, right?